components/Layout/Footer.tsx

```diff
@@ -8,17 +8,20 @@ import { FOOTER_LINK_GROUPS } from '@/const/menu'
 import { CONFIG } from '@/const/meta'
 
 const LogoImage = '/images/logo-light.svg'
+const LogoLightImageThemedCoWAMM = '/images/logo-light-themed-cowamm.svg'
 const CURRENT_YEAR = new Date().getFullYear()
 
-const Wrapper = styled.footer<{ noMargin?: boolean }>`
+const Wrapper = styled.footer<{ noMargin?: boolean; isCoWAMM?: boolean }>`
```

Comment:
I don't see this scaling for many other themes, but for now is ok. Reason, is if we creat 3 more pages with custom footer for example, we don't want to pass 3 booleans